### PR TITLE
[Docs] [Dask on Ray] Specify version compatibility

### PR DESCRIPTION
**Pull request by rkooo567**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/util/dask/common.py [hunk 1]
```diff
@@ -7,7 +7,12 @@
 
 from dask.base import quote
 from dask.core import get as get_sync
-from dask.utils import apply
+
+try:
```
Comment (amogkam) [inline python/ray/util/dask/common.py:11]: We don't need this anymore. `from dask.utils import apply` should already be backwards compatible.

**Review by amogkam — APPROVED** [review]

File: python/ray/util/dask/common.py (hunk 1 above)
Comment (rkooo567) [reply python/ray/util/dask/common.py]: wait so we don't need this PR at all?

Comment (amogkam) [reply python/ray/util/dask/common.py]: Yes if we want to add to the docs. But no code change is needed.

**Comment by rkooo567** [comment]
Only doc failures should be related to this PR. Merging it.

**Pull request merged**